vitest.setup.ts

```diff
@@ -128,6 +128,12 @@ class NangoActionMock {
         const paginateInBody = ['post', 'put', 'patch'].includes(args.method.toLowerCase());
         const updatedBodyOrParams = paginateInBody ? (args.data as Record<string, any>) || {} : args.params || {};
 
+        if (args.paginate['limit']) {
+            const limitParameterName = args.paginate.limit_name_in_request!;
```

Comment:
The `!` came up because Front doesn't have the limit parameter configured `limit_name_in_request`. I was surprised those tests failed but traced it back to here. Here's `updatedBodyOrParams` printing in Front's tests:

```
stdout | integrations/front/tests/front-conversation.test.ts > front conversation tests > should output the action output that is expected
{ undefined: 100 }
```

Reply:
Thanks, fixed in https://github.com/NangoHQ/integration-templates/pull/161